### PR TITLE
Changing config, to see tests in sippy dashboard

### PR DESCRIPTION
**Pull request by shahsahil264**
@paigerube14 @chaitanyaenr

### CODE REVIEW

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by paigerube14** [comment]
@shahsahil264 did you have issues with rebasing? There's 0 file changes here

**Comment by shahsahil264** [comment]
> @shahsahil264 did you have issues with rebasing? There's 0 file changes here

Hey @paigerube14 , Nope, I was trying things out and just wanted to check if the generated-config is running properly so I tried changing to the original config

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml [hunk 1]
```diff
@@ -38,9 +34,11 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    test:
-    - chain: redhat-chaos-krkn-hub-tests
-    - chain: redhat-chaos-hog-scenarios
+    workflow: redhat-chaos-krkn-hub-tests
```
Comment (paigerube14) [inline ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml:37]: you can only specify 1 workflow per test section

Comment (paigerube14) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml:37]: and a workflow can't be under tests (line 39) only ref or chains

Comment (shahsahil264) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml:37]: I saw workflow being used under test in the [periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm] file...so I thought I'll try it

Comment (paigerube14) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml:37]: yeh but you list 2 of them. can only have 1

Comment (shahsahil264) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml:37]: Yes yes....I ran the make ci-operator-config job and it removed one of the workflow

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Pull request closed without merging**